### PR TITLE
Remove deprecated parameters

### PR DESCRIPTION
**Pull request by cschwede**

### CODE REVIEW

File: pkg/controller/embercsi/controller.go [hunk 1]
```diff
@@ -95,7 +95,6 @@ func getControllerContainers(ecsi *embercsiv1alpha1.EmberCSI) []corev1.Container
 		args := []string{
 			"--v=5",
 			"--csi-address=/csi-data/csi.sock",
-			fmt.Sprintf("%s%s", "--provisioner=", GetPluginDomainName(ecsi.Name)),
```
Comment (Akrog) [inline pkg/controller/embercsi/controller.go]: Won't this break deployments for older OCP/K8s where it is needed?

File: pkg/controller/embercsi/controller.go [hunk 2]
```diff
@@ -76,7 +76,7 @@ func getControllerContainers(ecsi *embercsiv1alpha1.EmberCSI) []corev1.Container
 			Image: Conf.Sidecars[Cluster].Attacher,
 			Args: []string{"--v=5",
 				"--csi-address=/csi-data/csi.sock",
-				"--connection-timeout=120s",
```
Comment (Akrog) [inline pkg/controller/embercsi/controller.go]: External-attacher v2.0 only works for K8s v1.14 and later, so we would need to maintain the old parameter for those versions  right?

File: pkg/controller/embercsi/controller.go (hunk 1 above)
Comment (cschwede) [reply pkg/controller/embercsi/controller.go:98]: This has been deprecated in 1.0.1 of the provisioner: https://github.com/kubernetes-csi/external-provisioner/blob/master/CHANGELOG/CHANGELOG-1.1.md

And 1.1 is the lowest sidecar for k8s >= 1.13 / OCP >= 4.1. Should be ok therefore?

File: pkg/controller/embercsi/controller.go (hunk 2 above)
Comment (cschwede) [reply pkg/controller/embercsi/controller.go:79]: Right, but since time has passed by I think this should be safe today. Wdyt?

Comment (cschwede) [reply pkg/controller/embercsi/controller.go]: Nevermind, this has been replaced with the general timeout setting, dropped the commit.

**Review by Akrog — APPROVED** [review]
LGTM

**Pull request merged**